src/components/TradingTerminal/StrategyPanel/StrategyPanel.js

```diff
@@ -41,7 +42,7 @@ import "./StrategyPanel.scss";
  * @returns {JSX.Element} Strategy panel element.
  */
 const StrategyPanel = (props) => {
-  const { symbolData } = props;
+  const { symbolData, maxLeverage } = props;
```

Comment:
Why not using the maxLeverage property from symbolData instead of  all the custom get functions?

Reply:
@cwagner22  because when ever the symbol changes, I need to set leverage according to that specific symbol,

Reply:
Well, the component will re-render when symbolData changes so there is no need for that.

Reply:
@cwagner22  yeah, I just noticed that, fixed and pushed.